modules/core/core-scheduler/src/main/java/com/enonic/xp/impl/scheduler/ScheduleAuditLogSupportImpl.java

```diff
@@ -154,6 +154,11 @@ private void addResult( final PropertySet targetSet, final ScheduledJob job )
             targetSet.setInstant( ScheduledJobPropertyNames.LAST_RUN, job.getLastRun() );
         }
 
+        if ( job.getLastTaskId() != null )
```

Comment:
why would we need it in audit log? (actually lastRun is also confusing)

Reply:
why not? It may be useful for somebody